net-libs/libvncserver/libvncserver-0.9.12-r1.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit cmake-utils
+
+MY_PN="LibVNCServer"
+
+DESCRIPTION="library for creating vnc servers"
+HOMEPAGE="https://libvnc.github.io/"
+SRC_URI="https://github.com/LibVNC/${PN}/archive/${MY_PN}-${PV}.tar.gz"
+
+LICENSE="GPL-2"
+# No sub slot wanted (yet), see #578958
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~x86-fbsd ~amd64-linux ~x86-linux"
+IUSE="+24bpp gcrypt gnutls ipv6 +jpeg libressl lzo +png sasl ssl systemd +threads +zlib"
+REQUIRED_USE="!gnutls? ( ssl? ( threads ) ) png? ( zlib )"
+
+DEPEND="
+	gcrypt? ( >=dev-libs/libgcrypt-1.5.3:0= )
+	gnutls? (
+		>=net-libs/gnutls-2.12.23-r6:0=
+		>=dev-libs/libgcrypt-1.5.3:0=
```

Comment:
can you check if this dependency is correct? from the root `CMakeLists.txt` at least, I could not make the connection.

Reply:
This is still needed:
```
/var/tmp/portage/net-libs/libvncserver-0.9.12/work/libvncserver-LibVNCServer-0.9.12/common/rfbcrypto_gnutls.c:25:10: fatal error: gcrypt.h: No such file or directory
 #include <gcrypt.h>
          ^~~~~~~~~~
```
https://bugs.gentoo.org/420803

Reply:
I created PR fixing this: https://github.com/LibVNC/libvncserver/pull/306